src/GraphQL.NewtonsoftJson/StringExtensions.cs

```diff
@@ -41,15 +51,39 @@ public static Inputs ToInputs(this JObject obj)
         /// <returns>Returns a <c>null</c> if the object cannot be converted into a dictionary.</returns>
```

Comment:
Is it still true?

Reply:
It is as true or false as it was before (lol).

Reply:
It looked like invalid json would throw while a different type of json element would return null. I didn’t change that functionality.